### PR TITLE
Align pathfinder versions to what is production: latest

### PR DESCRIPTION
**Pull request by JacqueGM**
Closes https://github.com/CirclesUBI/circles-docker/issues/78

### CODE REVIEW

**Review by llunaCreixent — COMMENTED** [review]
when doing `make down`:
```
docker compose -f docker-compose.yml -f docker-compose.relayer-pull.yml -f docker-compose.api-pull.yml -f docker-compose.pathfinder-pull.yml -p circles down --remove-orphans --volumes
validating /.../CirclesUBI/circles-docker/docker-compose.pathfinder-pull.yml: services.indexer-db-init.depends_on.0 must be a string
make: *** [Makefile:63: down] Error 15
```

**Comment by JacqueGM** [comment]
it should be fine now - bad formatting 😶‍

File: docker-compose.pathfinder-pull.yml [hunk 1]
```diff
@@ -33,23 +35,22 @@ services:
...
-    build:
-      dockerfile: ${PWD}/blockchain-indexer/Dockerfile
-      context:    ${PWD}/blockchain-indexer
+    image: circlesubi/blockchain-indexer:latest 
     expose:
       - 8675
     depends_on:
-      indexer-db-init:
-        condition: service_healthy
+      - indexer-db-init:
```
Comment (llunaCreixent) [inline docker-compose.pathfinder-pull.yml:52]: `make down` does not work unless you remove the dash you added here. Same for the line 38

**Review by llunaCreixent — COMMENTED** [review]
The error reported persists

**Comment by llunaCreixent** [comment]
https://github.com/compose-spec/compose-spec/blob/master/spec.md#depends_on

**Review by juanenrisley — APPROVED** [review]
👍🏿

**Review by llunaCreixent — APPROVED** [review]
core tests passing

**Pull request merged**